src/StephanGroen/Tesla/Tesla.php

```diff
@@ -89,6 +89,11 @@ public function wakeUp() : array
         return $this->sendRequest('/wake_up', [], 'POST')['response'];
     }
 
+    public function startSoftwareUpdate() : array
+    {
+        return $this->sendRequest('/command/schedule_software_update', [ 'offset_sec' => 0 ], 'POST')['response'];
+    }
+
```

Comment:
The offset seconds should be able to be set as a variable and default to 0 (immediate), to allow for delayed installations.

Reply:
Good idea! Added a new commit with this change.